rag_app_sample_code/A_POC_app/pdf_uc_volume/04_create_evaluation_set.py

```diff
@@ -108,6 +108,20 @@
     F.col("is_correct"),
 ))
 
+# COMMAND ----------
+# MAGIC %md
+# MAGIC ## Ingest chats into the Review App to be reviewed 
+
+# COMMAND ----------
+
+from databricks.agents import enable_trace_reviews
+
+request_id_list = requests_with_feedback_df.select(F.col("request_id")).rdd.flatMap(lambda x: x).collect()
```

Comment:
should we be using the `request_log_df` instead of the `requests_with_feedback_df` here? it would only make sense to review request logs.

Reply:
Yeah, that makes sense. Fixed!